### DISCUSSION TITLE
Workflow: How do I verify that a field is empty

### DISCUSSION THREAD

**Discussion by juergen-de** [OrchardCMS/OrchardCore#8137] [Q&A]
I used the form/ workflow example for a message form and it works fine.
What I would like to do though is to verify that a field remains empty and I do not see where and how to do that.

**Comment by Skrypt** [comment]
if/else task.

**Reply by juergen-de** [reply]
Thank you, I did that, the if/else task requires a Javascript expression. How can I find out what the javascript expression has to look like? In the Send Email I can access {{ Request.Form.Name }} for example.

**Reply by Skrypt** [reply]
Here you will find a complete walkthrough to get you going.

http://www.ideliverable.com/blog/orchard-core-workflows-walkthrough-content-approval

Else, everything is documented https://docs.orchardcore.net/en/latest/docs/topics/workflows/

**Reply by juergen-de** [reply]
thanks a lot, thats a lot of information there.
